src/wp-includes/kses.php

```diff
@@ -2467,10 +2468,11 @@ function safecss_filter_attr( $css, $deprecated = '' ) {
 		}
 
 		if ( $found ) {
-			// Allow CSS calc().
-			$css_test_string = preg_replace( '/calc\(((?:\([^()]*\)?|[^()])*)\)/', '', $css_test_string );
+			// Allow some CSS functions.
+			$css_test_string = preg_replace( '/\b(?:calc|min|max|minmax|clamp)\(((?:\([^()]*\)?|[^()])*)\)/', '', $css_test_string );
+
 			// Allow CSS var().
-			$css_test_string = preg_replace( '/\(?var\(--[a-zA-Z0-9_-]*\)/', '', $css_test_string );
+			$css_test_string = preg_replace( '/\(?var\(--[\w|%\-\()[\]\,\s]*\)/', '', $css_test_string );
```

Comment:
Probably better to just use something like `[^()]` as in the regex above instead of having to include so many characters in the `[]` group.

Also, CSS identifiers may start with a single `-`.

<img width="863" alt="Screen Shot 2022-09-08 at 10 11 46" src="https://user-images.githubusercontent.com/612155/189006231-6a83cf6b-0af1-445a-8343-b4253d60ef43.png">

https://www.w3.org/TR/css-syntax-3/#typedef-ident-token